web/src/main/java/org/springframework/security/web/authentication/LoginUrlAuthenticationEntryPoint.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004, 2005, 2006 Acegi Technology Pty Limited
```

Comment:
It appears this update is not about improving diamond operator usage. If you want to keep it in, please do two things:

1. Leave any Acegi copyrights as-is
2. Move this to a separate commit for easier backporting

Reply:
I think I keep Acegi copyrights and just remove redundant keyword